src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php

```diff
@@ -47,59 +47,152 @@ class StockType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('allow_ordering_oos', SwitchType::class)
-            ->add('stock_management', SwitchType::class)
+            ->add('stock_management', SwitchType::class, [
+                'label' => $this->trans(
+                    'Enable stock management',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'required' => false,
+            ])
+            ->add('pack_stock_management', ChoiceType::class, [
+                'label' => $this->trans(
+                    'Default pack stock management',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'When selling packs of products, how do you want your stock to be calculated?',
+                    'Admin.Shopparameters.Help'
+                ),
+                'choices' => [
+                    'Decrement pack only.' => 0,
```

Comment:
I think you should use const here `PrestaShop\PrestaShop\Core\Domain\Product\Pack\ValueObject\PackStockType::STOCK_TYPE_PACK_ONLY`